docs/user-guide/docker-cli-to-kubectl.md

```diff
@@ -0,0 +1,252 @@
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<h1>PLEASE NOTE: This document applies to the HEAD of the source
+tree only. If you are using a released version of Kubernetes, you almost
+certainly want the docs that go with that version.</h1>
+
+<strong>Documentation for specific releases can be found at
+[releases.k8s.io](http://releases.k8s.io).</strong>
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+# kubectl for docker users
+
+In this doc, we introduce the kubernetes command line to for interacting with the api to docker-cli users. The tool, kubectl, is designed to be familiar to docker-cli users but there are a few necessary differences. Each section of this doc highlights a docker subcommand explains the kubectl equivalent.
+
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [kubectl for docker users](#kubectl-for-docker-users)
+      - [docker run](#docker-run)
+      - [docker ps](#docker-ps)
+      - [docker exec](#docker-exec)
+      - [docker logs](#docker-logs)
+      - [docker stop && docker rm](#docker-stop-&&-docker-rm)
+      - [docker login](#docker-login)
+      - [docker version](#docker-version)
+      - [docker info](#docker-info)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+#### docker run
+
+How do I run an nginx container and expose it to the world? Checkout [kubectl run](kubectl/kubectl_run.md).
+
+With docker:
+```
+$ docker run -d --name nginx-app -p 80:80 nginx
+a9ec34d9878748d2f33dc20cb25c714ff21da8d40558b45bfaec9955859075d0
+$ docker ps
+CONTAINER ID        IMAGE               COMMAND                CREATED             STATUS              PORTS                         NAMES
+a9ec34d98787        nginx               "nginx -g 'daemon of   2 seconds ago       Up 2 seconds        0.0.0.0:80->80/tcp, 443/tcp   nginx-app 
+```
+
+With kubectl:
+```
+# start the pod running nginx
+$ kubectl run --image=nginx nginx-app
+CONTROLLER   CONTAINER(S)   IMAGE(S)   SELECTOR        REPLICAS
+nginx-app    nginx-app      nginx      run=nginx-app   1
+# expose a port through with a
+$ kubectl expose rc nginx-app --port=80 --name=nginx-http
+NAME         LABELS          SELECTOR        IP(S)     PORT(S)
+nginx-http   run=nginx-app   run=nginx-app             80/TCP
+```
+
+With kubectl, we create a [replication controller](replication-controller.md) which will make sure that N pods are running nginx (where in is the number of replicas stated in the spec, which defaults to 1). We also create a [service](services.md) with a selector that matches the replication controller selector. This will configure the kube-proxy load balancers to target pods created by that replication controller on the specified port.
```

Comment:
I don't think you need to explain replication controllers and services here. See docs/user-guide/quick-start.md, which we should also link to here.